README.md

```diff
@@ -14,6 +14,19 @@ This program is released as open source software under the terms of [MIT License
 
 # Installing
 
+* Install the [DotNet SDK](https://www.microsoft.com/net/learn/get-started/)
+* `cd $BIONITIO_CODE/bionitio`
+
+## Build an executable on OS X, and then run
```

Comment:
Should this mention that `dotnet run` can be used to run the code without first compiling it? Compilation is only really needed if you want a self-contained binary

Reply:
Yes, that is fine with me. Feel free to amend as you like.

It would also be nice to have instructions about how to install the program into another location on the computer. I tried to make a self-contained binary executable, but I'm not sure whether certain dll files need to be copied as well.